http-generator-nima/src/main/java/io/avaje/http/generator/helidon/nima/ControllerWriter.java

```diff
@@ -126,30 +114,10 @@ private void writeClassStart() {
       writer.append("    this.validator = validator;").eol();
     }
     if (useJsonB) {
-      for (UType type : jsonTypes.values()) {
-        writer.append("    this.%sJsonType = jsonB.type(", type.shortName());
-        writeJsonbType(type);
+      for (final UType type : jsonTypes.values()) {
+        JsonBUtil.writeJsonbType(type, writer);
```

Comment:
nice

Reply:
It's the little things in life that mean the most